parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -186,13 +175,6 @@ write_files:
   content: !!binary |
     {{GetVariableProperty "cloudInitData" "healthMonitorScript"}}
 
-- path: /etc/systemd/system/kubelet-monitor.service
```

Comment:
never used, would actually kill kubelet since 1.22 (?) because the port changed and this would fail reliably every few seconds